src/protojure/pedestal/core.clj

```diff
@@ -146,11 +146,13 @@
   "Flushes and closes the response channel"
   [^HttpServerExchange exchange ^StreamSinkChannel ch]
   (log/debug "closing output channel" exchange)
-  (.shutdownWrites ch)
-  (loop []
-    (if (.flush ch)
-      (.close ch)
-      (recur)))
+  (try
+    (.shutdownWrites ch)
+    (loop []
+      (if (.flush ch)
+        (.close ch)
+        (recur)))
+    (catch Exception e))
```

Comment:
If this .close method can only throw a single Exception, probably better to restrict the catch to that type?

Reply:
The question then, though, is what would we do about it?  We wouldn't want to do something like (System/exit) because this is a library.  Not catching the error means we may skip other cleanup, such as the unsubscribe that occurs after this function.  The rationale is that we are on our way out the door anyway, just try to clean up as best we can.

Reply:
Fixed